hr_job_category/tests/test_hr_job_categories.py

```diff
@@ -61,7 +61,7 @@ def test_write_computes_with_normal_args(self):
 
         # Check if job2 categories are written to the employee
         self.contract_id.write({"job_id": self.job_2_id.id})
-        self.contract_id.flush()
+        self.contract_id.invalidate_recordset()
```

Comment:
`flush_recordset` perhaps? although it seems useless, we might be able to remove this line